pkg/cmd/gitprovider/add.go

```diff
@@ -18,6 +21,7 @@ var GitProviderAddCmd = &cobra.Command{
 	Use:     "add",
```

Comment:
Usage description should now be `add [GIT_PROVIDER_ID]`

Reply:
done